constants/Dictionary.ts

```diff
@@ -2,304 +2,402 @@ import * as Localization from 'expo-localization';
+        bestBefore: 'consúmase antes de',
+        successfullyAdded: 'agregado con éxito',
+        letsContinueWithOtherPerishableGood:
+            'continuemos con otro bien perecedero',
+        cancel: 'cancelar',
+        okay: 'Bien',
+        new: 'Nuevo',
+        chooseAPhoto: 'elige una foto',
+        expirationDate: 'fecha de expiración',
+        pickedLocation: 'ubicación elegida',
+        all: 'Mío',
+        signUp: 'regístrate',
+        name: 'nombre',
+        emailAddress: 'dirección de correo electrónico',
+        password: 'contraseña',
+        confirmPassword: 'confirmar contraseña',
+        submit: 'enviar',
+        youHaveSuccessfullysignedUp: 'te has registrado correctamente',
+        aProblemOccurredWhileCommunicatingWithTheServer:
+            'se produjo un problema al comunicarse con el servidor',
+        inOrderToMarkAsAvailableYouNeedToSignIn:
+            'para marcar como disponible, debe iniciar sesión',
+        signIn: 'iniciar sesión',
+        or: 'o',
+        pleaseEnterAName: 'por favor ingrese un nombre',
+        pleaseEnterAPassword: 'por favor ingrese una contraseña',
+        passwordMustBeAtLeast6Characters:
+            'la contraseña debe tener al menos 6 caracteres',
+        confirmPasswordMustBeEqualToPassword:
+            'confirmar la contraseña debe ser igual a la contraseña',
+        pleaseEnterAnEmailAddress:
+            'por favor ingrese una dirección de correo electrónico',
+        pleaseEnterAValidEmailAddress:
+            'por favor ingrese una dirección de correo electrónico válida',
+        nearby: 'Cercano',
+        meter: 'metro',
+        showMyNeed: 'mostrar mi interés',
+        statusOfMyRequest: 'estado de mi solicitud',
+        loading: 'cargando',
+        inOrderToShowYourNeedYouHaveToSignIn:
+            'para mostrar su interés, debe iniciar sesión',
+        lookWhoRequestedThis: 'alguien solicitó esto',
+        approve: 'aprobar',
+        goToSignIn: 'ir a iniciar sesión',
+        goToSignUp: 'ir a registrarse',
+        leaveAMessage: 'deja un mensaje',
+        egBreadMilkOrEggs: 'por ejemplo, pan, leche o huevos',
+        yourRequestHasNotYetBeenApproved:
+            'su solicitud aún no ha sido aprobada',
+        yourRequestHasAlreadyBeenApproved: 'su solicitud ya ha sido aprobada',
+        username: 'nombre de usuario',
+        address: 'dirección',
+        reply: 'respuesta',
+        approved: 'aprobada',
+        pickADate: 'selecciona una fecha',
+        confirm: 'confirmar',
+        unsupportedStatus: 'estado no soportado',
+        unsupportedContent: 'contenido no soportado',
+        yourGoodsAreNotFound: 'sus bienes no se encuentran',
+        noGoodsAreAvailable: 'no hay bienes disponibles',
+        resultSetIsEmpty: 'el conjunto de resultados está vacío',
+        pleaseEnterAnItem: 'por favor ingrese un item',
+        expiryMustNotBeAPastDate: 'la expiración no debe ser una fecha pasada',
+    },
```

Comment:
I would prefer to keep spaces because this makes it easy to see key-value pairs for all languages. You know what I mean? Like a table with two columns and their values exactly below each other.

Reply:
Yup sure. This was not intentional just the code formatter(prettier). I'll fix it. Thanks!

Reply:
Have you managed to make any progress on this?

Reply:
Yes, I've just pushed the code. Please check and tell me if something is left?